src/Hazelcast.Net.Tests/Networking/NetworkAddressTests.cs

```diff
@@ -29,6 +29,9 @@ namespace Hazelcast.Tests.Networking
     public class NetworkAddressTests
     {
         [TestCase("127.0.0.1", true, "127.0.0.1:0")]
+        [TestCase(" 127.0.0.1", true, "127.0.0.1:0")]
```

Comment:
Does port 0 means we will use port 5701 for connection? Asking this because that is our default port if none is provided.

Reply:
This is just to parse network address correctly independently from config. Server ports come up during connection establishment. 
For reference : https://github.com/hazelcast/hazelcast-csharp-client/blob/bbae59bea5cc254d7779affaa0dfb8e50a8704dd/src/Hazelcast.Net/Networking/AddressProvider.cs#L186